apps/files_external/lib/Controller/StoragesController.php

```diff
@@ -317,37 +317,14 @@ protected function updateStorageStatus(StorageConfig &$storage, $testOnly = true
 	 * @return DataResponse
 	 */
 	public function index() {
-		$storages = $this->formatStoragesForUI($this->service->getStorages());
+		$storages = array_map(fn ($storage) => $storage->jsonSerialize(true), $this->service->getStorages());
```

Comment:
```suggestion
		$storages = array_map(static fn ($storage) => $storage->jsonSerialize(true), $this->service->getStorages());
```
We can avoid to bind `$this`  in this callback

Reply:
Done